db/silkworm/db/access_layer.cpp

```diff
@@ -45,22 +45,29 @@ std::optional<BlockHeader> read_header(lmdb::Transaction& txn, uint64_t block_nu
 
 // TG ReadTransactions
 static std::vector<Transaction> read_transactions(lmdb::Transaction& txn, uint64_t base_id, uint64_t count) {
+    if (!count) {
+        return {};
+    }
+
+    auto table{txn.open(table::kEthTx)};
+    return read_transactions(table, base_id, count);
+}
+
+std::vector<Transaction> read_transactions(std::unique_ptr<lmdb::Table>& txn_table, uint64_t base_id, uint64_t count) {
```

Comment:
Since this method doesn't assume ownership of txn_table, it's simpler and faster to just pass `lmdb::Table& txn_table ` instead of `std::unique_ptr<lmdb::Table>& txn_table`